### ISSUE TITLE
Unable to attach Datascript to VS with enableEVH mode

### ISSUE THREAD

**Issue by 6547709** [vmware/load-balancer-and-ingress-services-for-kubernetes#457]
# Problem Description
   When the enableEVH option is 'true' in AKO, the datascript in HostRule cannot be loaded into VS;
   However, httpPolicySet is normal with EVH mode,datascript normal with SNI mode.
# Environmental information
    * vSphere with Tanzu 7.0U2a
    * Guest Cluster: 1.20.2
    * AKO: 1.4.2
    * AVI: 20.1.5
# hostrule yaml
```
apiVersion: ako.vmware.com/v1alpha1
kind: HostRule
metadata:
  name: avi-hello-kubernetes-hostrule
spec:
  virtualhost:
    fqdn: hello.avi.corp.local
    enableVirtualHost: true
    httpPolicy:
      policySets:
      - corp-httppolicy-rewrite-api
      overwrite: false
    tls: # optional
      sslKeyCertificate:
        name: dc03-corp-local
        type: ref
      sslProfile: System-Standard
      termination: edge
    datascripts:
    - corp-rewrite-api
    wafPolicy: dc03-vds-ako-waf-policy
    applicationProfile: dc03-vds-ako-http
```

**Comment by zm1990s** [comment]
It seems AKO 1.5.1 has fixed this issue.

**Comment by akshayhavile** [comment]
closed #457 via #475
